__tests__/integration.test.js

```diff
@@ -44,3 +44,40 @@ describe("/api", () => {
 			});
 	});
 });
+
+describe("/api/articles/:article_id", () => {
+	test("GET:200 sends the single article with specified id to the client", () => {
+		return request(app)
+			.get("/api/articles/3")
+			.expect(200)
+			.then(({ body: { article } }) => {
+				expect(article).toMatchObject({
+					article_id: 3,
+					title: "Eight pug gifs that remind me of mitch",
+					topic: "mitch",
+					author: "icellusedkars",
+					body: "some gifs",
+					created_at: "2020-11-03T09:12:00.000Z",
+					votes: 0,
+					article_img_url:
+						"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+				});
```

Comment:
Brilliant use of `toMatchObject` for a flexible test you won't have to change if you were to add extra keys to this response in the future 👀